CHANGELOG.md

```diff
@@ -8,6 +8,15 @@
 ### Features
 - Add grants to materializations ([dbt-labs/dbt-spark#366](https://github.com/dbt-labs/dbt-spark/issues/366), [dbt-labs/dbt-spark#381](https://github.com/dbt-labs/dbt-spark/pull/381))
 - Add `connection_parameters` for databricks-sql-connector connection parameters ([#135](https://github.com/databricks/dbt-databricks/pull/135))
+    - This can be used to customize the connection by setting the additional parameters.
```

Comment:
Nit, change to: "This can be used to customize the connection by setting **REMOVE**the**REMOVE** additional parameters